compiler/luci/pass/src/RemoveRedundantTranspose.test.cpp

```diff
@@ -105,6 +114,112 @@ void create_redundunt_transpose(loco::Graph *g, const std::vector<int32_t> &perm
   auto graph_output = g->outputs()->create();
   output->index(graph_output->index());
 }
+/**
+ *  BEFORE
+ *            |
+ *      [CircleNode]       [CircleConst]
+ *               \           /
+ *[CircleConst] [CircleTranspose] [CircleConst]
+ *        \          / \              /
+ *  [CircleTranspose] [CircleTranspose]
+ *          |                |
+ *     [CircleNode]     [CircleNode]
+ *
+ *  AFTER
+ * Type 1
+ *            |
+ *       [CircleNode]
+ *           /  \       Remove all transpose
+ * [CircleNode][CircleNode]
+ * Type 2
```

Comment:
Please add an empty line between ` * [CircleNode][CircleNode]` and ` * Type 2`

Reply:
Okay I will fix it.